Certified Sail Extension (CSE) Examples/cse/mapField/v1/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+
+<head>
```

Comment:
Indent on `<head>` & `<body>` tags?

Reply:
I think it's fine (and I personally prefer) to not indent since there's no functional need to do so and it just results in the entire block being bumped by 1 tab. If you have strong feelings about it I can bump it over though.